benchmarks/TF/linux-x86_64.cmake

```diff
@@ -20,6 +20,7 @@ set(LINUX_X86_64_CASCADELAKE_CPU_COMPILATION_FLAGS
   "--iree-input-type=mhlo"
   "--iree-llvm-target-cpu=cascadelake"
   "--iree-llvm-target-triple=x86_64-unknown-linux-gnu"
+  "--iree-llvmcpu-reassociate-fp-reductions=true"
```

Comment:
Sorry that I did not describe it clearly. There are two categories of benchmark suites. One is default-flags and the other is experimental-flags. We only set target related flags to `default-flags`. Setting the flag here would propagate toe all the benchmark suites.

Can we add `--iree-llvmcpu-reassociate-fp-reductions=true` to experimental-flags benchmark suites only?

Reply:
True, but the experimental path on benchmarks is used for MMT4D performance tracking. Totally understand numerical issues are something we need to have a handle on. But this is probably something we should have a flag, but we just turn it on by default cause that is kind of what people expect. If we see numerical issues (or are generating golden values) we could turn this flag off.